addon/ng2/blueprints/component/files/__path__/__name__.component.ts

```diff
@@ -0,0 +1,16 @@
+import {Component} from 'angular2/core';
+
+@Component({
+  moduleId: __moduleName,
+  selector: '<%= dasherizedModuleName %>',
+  templateUrl: '<%= dasherizedModuleName %>.component.html',
+  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+})
+export class <%= classifiedModuleName %>Component {
```

Comment:
I'd suggest to implement the `OnInit` interface if we're defining `ngOnInit` method.